app/lib/benchmarks_fixture.rb

```diff
@@ -0,0 +1,35 @@
+class InvalidParameter < Exception; end
+class OutOfBounds < Exception; end
```

Comment:
I haven't yet researched for stock exceptions to fill these roles

Reply:
`ArgumentError` and `RangeError` potentially: https://ruby-doc.org/core-2.5.1/Exception.html